src/main/scala/clump/ClumpSource.scala

```diff
@@ -16,14 +15,30 @@ class ClumpSource[T, U](val fetch: Set[T] => Future[Map[T, U]], val maxBatchSize
 
   def get(input: T): Clump[U] =
     new ClumpFetch(input, ClumpContext().fetcherFor(this))
+
+  def maxBatchSize(int: Int) =
```

Comment:
The ```maxBatchSize``` is now defined using this method. This change makes the ClumpSource's methods simpler and I also like the result for the client code:

New
```scala
ClumpSource(service.fetchTracks)(_.identifier).maxBatchSize(2)
```

Old
```scala
ClumpSource(service.fetchTracks, maxBatchSize = 2)(_.identifier)
```